apps/client/src/pages/CheckerPage.js

```diff
@@ -66,6 +67,28 @@ const CheckerPage = ({ checker, topic, resetChecker }) => {
 
   const isFinished = (component) => isActive(component, true);
 
+  /**
+   * Set the questionIndex the next questionId, previous questionId, or the given id.
+   *
+   * @param { int | ('next'|'prev') } value - This van be, 'next, prev or a int`
+   */
+  const goToQuestion = (value) => {
+    const newIndex = Number.isInteger(value)
+      ? value
+      : value === "next"
+      ? questionIndex + 1
+      : value === "prev"
+      ? questionIndex - 1
+      : console.error(`${value} is not a int, 'next' or 'prev'`);
```

Comment:
I like it because of its simplicity. But can you add a comment `// Return the value if value isInteger()` on line 77?

Nitpicking, but can you change the error to: `goToQuestion(): ${value} is not an integer, 'next' or 'prev'`